src/k8s-engine/k3sHelper.ts

```diff
@@ -611,4 +611,23 @@ export default class K3sHelper extends events.EventEmitter {
 
     return contents;
   }
+
+  /**
+   * Delete state related to Kubernetes.  This will ensure that images are not
+   * deleted.
+   * @param execAsRoot A function to run commands on the VM as root.
+   */
+  async deleteKubeState(execAsRoot: (...args: string[]) => Promise<void>) {
+    const directories = [
+      '/var/lib/kubelet', // https://github.com/kubernetes/kubernetes/pull/86689
+      // We need to keep /var/lib/rancher/k3s/agent/containerd for the images.
+      '/var/lib/rancher/k3s/data',
+      '/var/lib/rancher/k3s/server',
```

Comment:
Should we delete `/var/lib/rancher/k3s/storage` as well? I believe this is the location used for PVC storage provided by our local-path-provisioner. I don't think the data can be recovered since the PVCs themselves are gone when you wipe the object store.

Please verify the path though; this is from memory.

Reply:
Yes, we should; it wasn't included because I checked against a freshly deployed VM, which didn't have that directory because there were no PVCs…

Reply:
Fixed in fc12446c30dc2ad165510ffa090eea3a119ca6b0 — checked the path after deploying wordpress (which uses PVCs) to the cluster.